.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: Continuous Integration
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  # See: https://github.community/t/duplicate-checks-on-push-and-pull-request-simultaneous-event/18012
+  push:
+    branches:
+      - main*
+  pull_request:
+    branches:
+      - main*
+
+jobs:
+  build:
+    name: Full Build (node ${{ matrix.node_version }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node_version: [14.x, 16.x]
```

Comment:
nodejs 18?

Reply:
technical debt: frontend build has a problem with node18 -> anyhow should adapt frontend in vue3